File: Backend/VestTour.Domain/Entities/ProductInStore.cs
using System;
using System.Collections.Generic;

namespace VestTour.Domain.Entities;

public partial class ProductInStore
{
    public int StoreId { get; set; }

    public int ProductId { get; set; }

    public int Quantity { get; set; }

    public virtual Product Product { get; set; } = null!;

    public virtual Store Store { get; set; } = null!;
}

File: Mobile/mobileapp/lib/models/product_in_store.dart
class ProductInStore {
  int? storeID;
  int? productID;
  int? quantity;

  ProductInStore({this.storeID, this.productID, this.quantity});

  ProductInStore.fromJson(Map<String, dynamic> json) {
    storeID = json['storeID'];
    productID = json['productID'];
    quantity = json['quantity'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['storeID'] = this.storeID;
    data['productID'] = this.productID;
    data['quantity'] = this.quantity;
    return data;
  }
}
